.github/renovate.json

```diff
@@ -8,5 +8,10 @@
     "groupName": "phoenix_view",
     "managers": ["mix"],
     "separateMajorMinor": false
-  }]
+  }],
+  "pip_requirements": {
```

Comment:
Shouldn't be any need for changes to this file, since renovate should run on all `requirements.txt` files by default.